backend/src/akvo/lumen/component/keycloak.clj

```diff
@@ -37,7 +53,7 @@
   (-> (client/get (format "%s/.well-known/openid-configuration" issuer))
       :body json/decode))
 
-(defn request-draft
+(defn request-headers
   "Create a partial request with json content type and a valid bearer token.
```

Comment:
I think this docstring should be tweaked a bit now when you return headers only?